main.nf

```diff
@@ -3451,7 +3451,7 @@ process ControlFreecVizSingle {
     script:
     """
     echo "Shaping CNV files to make sure we can assess significance"
-    LINEWIDTH=`head -1 ${cnvTumor}| wc -w`; awk 'NF=='$LINEWIDTH'{print}' ${cnvTumor} > TUMOR.CNVs
+    LINEWIDTH=`head -1 ${cnvTumor}| wc -w`; awk 'NF=='\$LINEWIDTH'{print}' ${cnvTumor} > TUMOR.CNVs
```

Comment:
I think it is easier to use this sentence:

awk '{print $NF}' ${cnvTumor} > TUMOR.CNVs

Reply:
Or this one instead?
```
awk '{print \$NF}' ${cnvTumor} > TUMOR.CNVs
```

Reply:
Actually this line only print the last column, don't we need to print all lines that have the right number of column?

Reply:
Actually, that line to reshape the CNVs is not needed. 

https://github.com/BoevaLab/FREEC/blob/master/scripts/assess_significance.R

Is already taking into consideration the shape of the input CNVs.